src/layouts/Layouts.jsx

```diff
@@ -1,11 +1,17 @@
 import React from 'react';
+import { useSession } from '@inrupt/solid-ui-react';
 import { NavBar } from '../components/NavBar';
+import { InactivityMessage } from '../components/Notification';
 
-const Layout = ({ children }) => (
-  <>
-    <NavBar />
-    {children}
-  </>
-);
+const Layout = ({ children }) => {
+  const { session } = useSession();
+  return (
+    <>
+      <NavBar />
+      {children}
+      {session.info.isLoggedIn && <InactivityMessage />}
```

Comment:
`<Footer />` also

Reply:
@leekahung not sure how'd you feel about this, but I also propose we rename this file to `Home.jsx`

This seems to be the layout for the entire app. (Especially after moving the login/logout button functionality into the `Navbar`.) After that is done this would replace `Login.jsx` for the home route.

```js
<Navbar />
< any route component the user has clicked/been directed to />
<Footer />
```

Reply:
I think we could keep this as Layouts.jsx.

Considering we'll probably move on to building dedicated pages from components, I think we can reserve Home.jsx for that.

Reply:
> I think we could keep this as Layouts.jsx.
> 
> Considering we'll probably move on to building dedicated pages from components, I think we can reserve Home.jsx for that.

Oh yes, sorry I'm dumb.  Not sure where my brain was at there.